src/test/compile-fail/issue-27576.rs

```diff
@@ -0,0 +1,14 @@
+trait Setup {
```

Comment:
Can you add a brief comment describing the issue, and change the filename to something like 'issue-27576-unchecked-bounds.rs'?

Reply:
I had messed up the issue number in the file name :sweat_smile: 
It's `issue-27675-unchecked-bounds.rs` now, with description.